r2-netty/src/main/java/com/linkedin/r2/netty/client/http/HttpChannelInitializer.java

```diff
@@ -86,7 +86,7 @@
     _maxInitialLineLength = maxInitialLineLength;
     _maxHeaderSize = maxHeaderSize;
     _maxChunkSize = maxChunkSize;
-    _maxContentLength = maxContentLength > Integer.MAX_VALUE ? Integer.MAX_VALUE : (int) maxContentLength;
+    _maxContentLength = maxContentLength;
```

Comment:
This is in Http2ChannelInitilizer too, should we fix that file as well?

Reply:
We can't fix it in Http2ChannelInitializer as Netty expects an integer value there and the one in Http2ChannelInitializer is not used in StreamHandler

Reply:
The one used in Http2ChannelInitializer is used for parent channel and it is not used to serve the same purpose in HttpChannelInitializer. The http2 equivalent of the same can be found @ Http2StreamChannelInitializer

Reply:
Got it, thanks